### PR TITLE
feat: added homepage

### PR DESCRIPTION
**Pull request by muskan-gupta-0803**
Added homepage
- created a quiz categories section

### CODE REVIEW

**Comment by sohan9819** [comment]
LGTM 👍

**Review by sohan9819 — APPROVED** [review]
LGTM 👍

File: index.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                    <img class="card-img br-sm" src="https://picsum.photos/250/150" alt=""/>
+                    <div class="card-badge">category</div>
+                    <div class="card-body">
+                    <h3 class="card-title">Title</h3>
+                    <small class="card-subtitle">5 Questions</small>
+                    <p class="card-description">Take this quiz to test youself! </p>
+                    </div>
+                </div>
+            </a>
+
```
Comment (raieegulhane) [inline index.html]: Remove white spaces

**Review by raieegulhane — COMMENTED** [review]
Good to merge

**Pull request merged**